h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.html

```diff
@@ -296,73 +258,41 @@
           </label>
 
         </div>
-        <div class="col-xs-1"></div>
       </div>
-      <div class="row form-group mb-0" *ngIf="form.get('clientNetwork').value">
-        <div class="col-xs offset-xs-6 text-right">
-
-          <label for="client-network-gateway">
-            Gateway
+      <div class="row form-group" *ngIf="form.get('clientNetwork').value">
+        <div class="col-xs-4 text-right">
+          <label for="client-network-routing">
+            Routing destination:Gateway
           </label>
-
         </div>
-        <div class="col-xs">
-
-          <label for="client-network-gateway"
-                 aria-haspopup="true"
-                 role="tooltip"
-                 class="form-control tooltip tooltip-validation tooltip-md tooltip-top-left"
-                 [class.invalid]="form.get('clientNetworkGateway').invalid && (form.get('clientNetworkGateway').dirty || form.get('clientNetworkGateway').touched)">
+        <div class="col-xs routing-conf col-xs-8">
+          <label for="client-network-routing" aria-haspopup="true" role="tooltip" class="form-control tooltip tooltip-validation tooltip-md tooltip-top-left"
+            [class.invalid]="form.get('clientNetworkRouting').invalid && (form.get('clientNetworkRouting').dirty || form.get('clientNetworkRouting').touched)">
 
-            <input class="form-control"
-                   id="client-network-gateway"
-                   type="text"
-                   placeholder="IP address"
-                   formControlName="clientNetworkGateway">
+            <input class="form-control" id="client-network-routing" type="text" placeholder="Comma delimited CIDRs" formControlName="clientNetworkRouting">
 
-            <span class="tooltip-content" *ngIf="form.get('clientNetworkGateway').hasError('required')">
-              Client network gateway cannot be empty
+            <span class="tooltip-content" *ngIf="form.get('clientNetworkRouting').hasError('required')">
+              Routing destination cannot be empty
             </span>
-
-            <span class="tooltip-content" *ngIf="form.get('clientNetworkGateway').hasError('pattern')">
-              Gateway address is not valid
+            <span class="tooltip-content" *ngIf="form.get('clientNetworkRouting').hasError('pattern')">
+              IP addresses are not valid,please input one or more routing destinations in a comma separated list, e.g. 10.1.0.0/16,10.2.0.0/16:10.0.0.1
```

Comment:
> IP addresses are not valid,please input ...

There's a space missing after the comma ("valid, please").

> e.g. 10.1.0.0/16,10.2.0.0/16:10.0.0.1

This content seems to be for the routing destinations field. Maybe it should exclude the ":10.0.0.1" from the example, since that goes in the other field?

Reply:
done.